neurodamus/lfp_manager.py

```diff
@@ -0,0 +1,95 @@
+import logging
+import numpy
+
+from .core import NeurodamusCore as Nd
+from .core.configuration import ConfigurationError
+
+
+class LFPManager:
+    """
+    Class handling the lfp functionality
+    """
+    def __init__(self):
+        self._lfp_file = None
```

Comment:
Do you think it would be easy to make this a bit more RAII, so that, if we have `LFPManager` instance (not None) is always has the handle to the file? Smtg like
```
def __init__(self, lfp_weights_file):
    try:
        self._lfp_file = h5py.File(lfp_weights_file, 'r')
    except IOError as e:
        raise ConfigurationError(f"Error opening LFP electrodes file: {e}")
    self._available_populations = ...
```
That way we fail early if there's any problem, and the methods are simpler

Reply:
Makes sense, however if we want to keep the construction of the LFPManager() in the GlobalCellManager constructor, we would need to pass around the path to the file and it doesnt feel right to me.

What i could do is to create the manager in the node.py, the same place where i call lfp_manager.load_lfp_config(lfp_weights_file, population_list)

But then it would fail at the same time than right now...

Otherwise we can merge this and see in the future